sites/feedback/ajax/page-feedback-en.html

```diff
@@ -0,0 +1,74 @@
+<div class="gc-pft row" data-wb-jsonmanager='{
+	"name": "feedback",
+	"extractor": [
+		{ "interface": "locationHref", "path": "submissionPage" },
+		{ "selector": "title", "path": "pageTitle" },
+		{ "selector": "html", "attr": "lang", "path": "language" },
+		{ "selector": "meta[name=\"dcterms.type\"]", "attr": "content", "path": "themeopt" },
+		{ "selector": "meta[name=\"dcterms.subject\"]", "attr": "content", "path": "sectionopt" },
+		{ "selector": "meta[name=\"author\"]", "attr": "content", "path": "institutionopt" }
+	]
+}'>
+	<div class="col-sm-10 col-md-9 col-lg-8">
+		<section class="well mrgn-bttm-0" aria-live="polite">
+			<h3 class="wb-inv">Page feedback</h3>
+			<form id="gc-pft-frm" action="https://pagesuccessemailqueue.azurewebsites.net/api/QueueProblemForm" method="post" class="wb-postback" data-wb-postback='{"success":".gc-pft-thnk", "content":"#gc-pft-frm"}'>
```

Comment:
```suggestion
			<form action="https://pagesuccessemailqueue.azurewebsites.net/api/QueueProblemForm" method="post" class="wb-postback" data-wb-postback='{"success":".gc-pft-thnk"}'>
```

You might be able to get away with scrapping the form ID and the ``content`` variable.

I didn't realize this while prepping #1916, but it looks like the postback plugin doesn't actually use a ``content`` variable anymore. Maybe it only existed when that plugin was provisional and got scrapped by the time it became stable.

As for the form ID, it looks like only that useless postback feature needed it. So it doesn't really serve a purpose anymore (AFAIK).